src/pages/404.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from "next/link";
+
+import Footer from "@/components/footer/Footer";
+import HomeTextBanner from "@/components/landingPage/HomeTextBanner";
+import NavBar from "@/components/navBar/NavBar";
+
+export default function Custom404() {
+  return (
+    <div className={`bg-custom-background text-custom-primary-text`}>
+      <main
+        id="main"
+        className="min-h-[calc(100dvh-122px)] flex w-full items-center"
```

Comment:
```suggestion
        className="min-h-[calc(100dvh-122px)] flex w-full items-center pt-[60px] pb-[60px]"
```

the padding-top help preserve the header space on really small screens.
the padding-bottom is to ensure centered vertical spacing.

I doubt that it is much useful here as the not found is the only content and doesn't take much vertical spacing.

Either ways, this works!

Reply:
Removing the padding styles had no significant effect on the page. So I chose to do that to reduce the code since it's unnecessary.

Reply:
Its not noticeable till vertical height is <422px